packages/token-provider/README.md

```diff
@@ -0,0 +1,11 @@
+# token-provider
```

Comment:
Can we get more infos on this library ? 
What is it ? How it works? What it is used for ? 😇